framework/Sources/VGSFramework/UIElements/Text Field/VGSTextField.swift

```diff
@@ -138,8 +138,12 @@ public class VGSTextField: UIView {
                                                                 views: views)
         NSLayoutConstraint.activate(verticalConstraint)
         
-        //delegate
-        textField.addSomeTarget(self, action: #selector(textFieldDidChange), for: .editingChanged)
+        textField.onEditingChange = { [weak self] in
+            if let strongSelf = self {
+                strongSelf.vgsCollector?.updateStatus(for: strongSelf)
+            }
+        }
+         //delegates
```

Comment:
````swift
textField.onEditingChange = { [weak self] in
    self?.vgsCollector?.updateStatus(for: strongSelf)
}
````
- more beautiful format

Reply:
**updateStatus**: takes required instance, not **self?**